.github/ISSUE_TEMPLATE/1.bug_report.yaml

```
@@ -85,6 +86,7 @@ body:
         - delete
         - env
```

Can we add export too?
```suggestion
        - env
        - export
```